Bio/SeqIO/SeqXmlIO.py

```diff
@@ -57,8 +57,13 @@ def __iter__(self):
         """Iterate over the records in the XML file."""
         record = None
         try:
+            flag = False
```

Comment:
``flag`` is too generic - I suggest ``empty`` as you've done in ``Bio/SeqIO/PdbIO.py`` (with the inverted values)